aep/general/0135/aep.md.j2

```diff
@@ -48,15 +78,7 @@ rpc DeleteBook(DeleteBookRequest) returns (google.protobuf.Empty) {
   that the existence of the resource has reached a steady-state and reading
   resource state returns a consistent response.
 
-
-
-The Delete method **should** succeed if and only if a resource was present and
-was successfully deleted. If the resource did not exist, the method **should**
-send a `NOT_FOUND` error.
-
-### Request message
-
-Delete methods implement a common request message pattern:
+Delete methods implement a common request pattern:
 
 ```proto
 message DeleteBookRequest {
```

Comment:
GH won't let me suggest on the right line, but `name` should be `path`, and `library.googleapis.com` should be (I think) `library.example.com`.

Same applies to other proto snippets below.

Reply:
Changed all "library.googleapis.com" to "library.example.com".

Which line number should have "name" -> "path"?

Reply:
Changed "google." to "library." everywhere.